src/main/java/com/watch/shop/app/service/WatchServiceImpl.java

```diff
@@ -0,0 +1,43 @@
+package com.watch.shop.app.service;
+
+import com.watch.shop.app.model.Watch;
+import com.watch.shop.app.model.WatchStore;
+
+import java.util.Comparator;
+import java.util.List;
+
+import static java.util.Objects.isNull;
+
+public class WatchServiceImpl implements WatchService {
+
+    private final WatchStore watchStore = WatchStore.getInstance();
+
+    public List<Watch> getWatches() {
+        return watchStore.getWatches();
+    }
+
+    public List<Watch> sortWatchesByParam(String param) {
+        if (isNull(param) || param.isBlank()) throw new IllegalArgumentException("Param cannot be empty");
```

Comment:
IF and FOR always should be with { }, because it's significally decrease readibility and increase bug producing

`if (smth > 34 || someMethod2(89) && ((yourMethod)) someMethod3();`
is very poorly to read, we don't see clearly where condition ends